index.js

```diff
@@ -13,7 +13,9 @@ try {
   logger.debug("Process was running %s seconds before agent was loaded.",
                process.uptime())
 
-  if (process.version && process.version.split('.')[1] < 6) {
+  var major =  process.version.split('.')[0];
+  var minor =  process.version.split('.')[1];
+  if (process.version && (major === 0) && (minor < 6)) {
```

Comment:
`minor` works because of it's implicit conversion, but `major` is doing a type and value equality, so it'll need to be converted to number.